dev/README.md

```diff
@@ -50,6 +52,9 @@ One error we ran into before was in code-blocks with HTML specifically, so let's
 ![An image from a vintage video synth](./static/some_art.png)
 Here is an image I made.
 
+Here is one with an external link
+![](https://sarahghp.com/vids/noodling/noodl06)
```

Comment:
Also, I almost forgot, we should link to something that is not mine. Do you have a suggestion for a stable remote link?

Reply:
We can use unsplash images, they don't get deleted

https://unsplash.com/photos/J4CeJunkE8M/download?ixid=MnwxMjA3fDB8MXxhbGx8MTR8fHx8fHwyfHwxNjc0NTYzNDA0&force=true&w=640

Reply:
Updated with pexels, since it is rights free. Thank you for the suggestion!